### PR TITLE
Add support page

### PR DESCRIPTION
**Pull request by jamesmaa**
image

cc @martholomew

### CODE REVIEW

File: ext/info.html [hunk 1]
```diff
@@ -66,11 +66,13 @@ <h2 id="links">Links</h2>
     <div class="settings-group">
         <div class="settings-item"><div class="settings-item-inner"><div class="settings-item-left"><div class="settings-item-label">
             <ul>
-                <li>Information and downloadable dictionaries: <a href="https://github.com/yomidevs/yomitan#yomitan" rel="noreferrer noopener">Homepage</a></li>
-                <li>Support and source code: <a href="https://github.com/yomidevs/yomitan" rel="noreferrer noopener">Github</a></li>
+                <li>Information and user guide: <a href="https://yomitan.wiki" rel="noreferrer noopener">Homepage</a></li>
+                <li>Free starter dictionaries: <a href="https://github.com/yomidevs/kaikki-to-yomitan/blob/master/downloads.md" rel="noreferrer noopener">Yomitan Wiktionary Dictionaries</a></li>
```
Comment (MarvNC) [inline ext/info.html:70]: Why not link https://yomitan.wiki/dictionaries/?

File: ext/support.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    <span tabindex="-1" id="content-scroll-focus"></span>
+
+    <h1>Support Yomitan ❤️</h1>
+
+    <h2>Here are some ways to support Yomitan:</h2>
+    <div class="settings-group">
+        <div class="settings-item"><div class="settings-item-inner"><div class="settings-item-left"><div class="settings-item-label">
+            <ul>
+                <li>Leave a review on the Firefox/Chrome/Edge store! <a href="https://addons.mozilla.org/en-US/firefox/addon/yomitan/">Firefox</a> | <a href="https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn">Chrome</a> | <a href="https://microsoftedge.microsoft.com/addons/detail/yomitan/idelnfbbmikgfiejhgmddlbkfgiifnnn">Edge</a></li>
```
Comment (MarvNC) [inline ext/support.html:33]:
```suggestion
                <li>Leave us a review on the Firefox/Chrome/Edge store! <a href="https://addons.mozilla.org/en-US/firefox/addon/yomitan/">Firefox</a> | <a href="https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn">Chrome</a> | <a href="https://microsoftedge.microsoft.com/addons/detail/yomitan/idelnfbbmikgfiejhgmddlbkfgiifnnn">Edge</a></li>
```

File: ext/support.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                <li>Report any bugs or feedback on our <a href="https://github.com/yomidevs/yomitan/issues/new/choose" rel="noreferrer noopener">Github Issues</a>.</li>
+                <li>Contribute to the Yomitan source code! Start by reading our <a href="https://github.com/yomidevs/yomitan/blob/master/CONTRIBUTING.md" rel="noreferrer noopener">Contribution Guidelines</a>.</li>
```
Comment (MarvNC) [inline ext/support.html:35]:
```suggestion
                <li>Contribute to the Yomitan extension! Start by reading our <a href="https://github.com/yomidevs/yomitan/blob/master/CONTRIBUTING.md" rel="noreferrer noopener">Contribution Guidelines</a>.</li>
```

Comment (brishtibheja) [reply ext/support.html:35]: How about "Yomitan Project"?

**Pull request merged**